crawl4ai/async_url_seeder.py

```
@@ -0,0 +1,1471 @@
+                        loc = loc_elem.text.strip() if loc_elem.text else ""
+                        if loc:
+                            regular_urls.append(loc)
+            except Exception as e:
+                self._log("error", "LXML parsing error for sitemap {url}: {error}",
+                          params={"url": url, "error": str(e)}, tag="URL_SEED")
+                return
+        else:  # Fallback to xml.etree.ElementTree
+            import xml.etree.ElementTree as ET
+            try:
+                # Parse the XML
+                root = ET.fromstring(data)
```

Ruff (0.14.0)

Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)